src/wp-includes/functions.php

```diff
@@ -6827,7 +6827,7 @@ function wp_timezone_choice( $selected_zone, $locale = null ) {
  * @see https://core.trac.wordpress.org/ticket/8497
  *
  * @param string $str Header comment to clean up.
- * @return string
+ * @return string Cleaned up header comment.
```

Comment:
The word "up" should be unnecessary:
```suggestion
 * @return string Cleaned header comment.
```

I do not have a clear idea of how to edit this further, but the `_cleanup_header_comment()` function cleans lines within a file header comment, especially if the comment has only one line.